### ISSUE TITLE
Estrutura do feedback que todo candidato receberá

### ISSUE THREAD

**Issue by rafaels88** [training-center/hiring-center#5]
O principal valor que este projeto pode entregar é o feedback. Por isso é importante que tenhamos um modelo que defina o mínimo que um feedback deve ter para que deixemos passar nenhum detalhe importante.

Quais pontos todo feedback deve ter?

**Comment by nicholasess** [comment]
Acredito que poderia ter tópicos e neles uma avaliação breve:

CV
- Seu cv é completo, mas percebemos que seu conhecimento na prática é bem diferente. Acredito que você deveria ser verdadeiro para quem está contratando, para não ter frustrações no dia a dia

Entrevista
- Procura saber mais sobre as tecnologias que foram proposta da vaga, percebemos que você sabia, mas a teoria deixou a desejar...

Test
- Aqui você foi excelente, fez todos os itens que eram necessário, mas adicionou itens que não eram necessários. Nós da empresa XPT, procuramos entregar as funcionalidades com o minimo de recurso para sermos efetivos ao que requerem. Mesmo com sprints temos prazos curtos por sermos uma empresa lider de mercado.

Você passou no test, iremos entrar em contato para a próxima fase.

--------

Mais ou menos assim

**Comment by rafaels88** [comment]
@nicholasess Desculpe a demora em responder aqui.

Gostei demais da sua proposta! Eu adicionaria dois sub-tópicos aos que você propôs, `Prós` e `Contras`, abaixo de cada tópico que você sugeriu, faz sentido? Apenas pra deixar a coisa mais estruturada mesmo. Me lembro de ter recebido um feedback da Thoughtworks assim e achei incrível. Ficou tudo bem explicado.

**Comment by nicholasess** [comment]
@rafaels88 sim, acredito que prós e contras fica mais democrático, assim o candidato consegue distinguir e melhorar só o que precisa. Quando se tem somente um texto, o candidato fica a merce da interpretação.
